apps/microtvm/zephyr/template_project/src/mlperftiny/tvmruntime.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdarg.h>
+#include <tvm/runtime/crt/error_codes.h>
+#include <unistd.h>
+
+#define MODEL_KWS 1
+#define MODEL_VWW 2
+#define MODEL_AD 3
+#define MODEL_IC 4
+
+extern const unsigned char g_wakeup_sequence[];
+extern size_t g_output_data_len;
+
+#if TARGET_MODEL == 3
+extern float* g_output_data;
+#else
+extern int8_t* g_output_data;
+#endif
+
+extern float g_quant_scale;
+extern int8_t g_quant_zero;
+
+void TVMRuntimeInit();
```

Comment:
add doxygen comments for these functions?

Reply:
thanks for catching this, I added documentation and fixed the header macros.